BatchCave.py

```diff
@@ -18,31 +26,41 @@
 
     **** UC Libraries Record Loads Working Group ****
 
-"""
+    """)
 
 def BrowseFiles():#open file explorer
-    root=Tk()
+    root=tk.Tk()
     root.withdraw()
     filenameOUT = askopenfilename(filetypes=[("MARC files","*.mrc"),("XML files","*.xml"),("All the files","*.*")], title="R.L.W.G Batch Edit -- select input file")
-    print type(filenameOUT)
+    print (type(filenameOUT))
     filename = filenameOUT
     
     root.destroy()
-    print '\n\nSelected file: \"' + re.sub('.*/(.*?)$', '\\1\"', filename) 
+    print ('\n\nSelected file: \"' + re.sub('.*/(.*?)$', '\\1\"', filename) )
 
     return filename
 
 class utilityFunctions:
 
+
     def listChangeScripts(self, BatchEdits):
-        
-        num = 0
-        ChangeScriptsDict = {}
-        for i in dir(BatchEdits)[:-2]:
-            num = num + 1
-            ChangeScriptsDict[num] = [i, ''.join(inspect.getargspec(getattr(BatchEdits, i))[3])]
+        num = range(len(dir(BatchEdits)[:-26]))
+
+            # if you want names and values as a dictionary:
+        args_dict = dict(zip(num , dir(BatchEdits)[:-26]))
+        #print(num)

```

Comment:
Are these `#print(...)` calls all debugging statements? Should we remove?

Reply:
Yeah. We should remove.